server.js

```diff
@@ -165,8 +165,8 @@ function initialise () {
   initialisei18n(app)
   initialiseCookies(app)
   initialiseTemplateEngine(app)
-  initialiseRoutes(app)
   initialisePublic(app)
+  initialiseRoutes(app)
```

Comment:
If it is imperative that this comes last it would be good to add a comment here to protect against future mishaps, as you've done for app/router.js.